handler.ts

```diff
@@ -1,25 +1,54 @@
+import * as S3 from 'aws-sdk/clients/s3';
+import * as SSM from "aws-sdk/clients/ssm";
+import { URL } from "url";
 import { Callback, Context, Handler, ScheduledEvent } from "aws-lambda";
-import { Employee } from "./domain/Employee";
 import { INversaryEvent } from "./domain/NversaryEvent";
-import {SlackConfiguration} from "./domain/SlackConfiguration";
-import {EmployeeRepositoryLocalImpl} from "./repository/EmployeeRepositoryLocalImpl";
+import { SlackConfiguration} from "./domain/SlackConfiguration";
+import { EmployeeRepositoryLocalImpl} from "./repository/EmployeeRepositoryLocalImpl";
 import { AnniversaryService } from "./service/AnniversaryService";
-import {CongratulationService} from "./service/CongratulationService";
-import {SlackService} from "./service/SlackService";
-import * as data from "./test/data/people_real.json";
+import { CongratulationService } from "./service/CongratulationService";
+import { SlackService } from "./service/SlackService";
 
-export const greeter: Handler = (event: INversaryEvent, context: Context, cb: Callback) => {
-  console.log("event: \n" + JSON.stringify(event, null, 2));
+interface Config {
+  slack: SlackConfiguration
+}
+
+/**
+ * Fetch configuration from SSM parameters.
+ */
+const fetchConfig = async (parameterName: string): Promise<Config> => {
+  console.info(`Fetch SSM parameter ${parameterName}`)
+  const ssm = new SSM();
+  return ssm.getParameter({Name: parameterName, WithDecryption: true}).promise()
+    .then((response) => JSON.parse(response.Parameter.Value))
+    .then((config) => ({
+      slack: new SlackConfiguration(config.slack.webhookUrl as string,
+        config.slack.channelId as string,
+        config.slack.appToken as string,
+        false)
```

Comment:
Instead of type casting we could do `String(config.slack.webhookUrl)` etc to ensure they are strings!

Reply:
That would convert the non-string values to string and probably cause more trouble than getting exceptions when non-string values are used as strings.